metals/src/main/scala/scala/meta/internal/metals/Messages.scala

```diff
@@ -160,6 +160,26 @@ class Messages(icons: Icons) {
       params
     }
   }
+  object BloopVersionChange {
+    def reconnect: MessageActionItem =
+      new MessageActionItem("Restart Bloop")
+    def notNow: MessageActionItem =
+      new MessageActionItem("Not now")
+    def params(version: String): ShowMessageRequestParams = {
+      val params = new ShowMessageRequestParams()
+      params.setMessage(
+        s"Bloop version was updated, do you want to kill the running server and start Bloop $version?"
```

Comment:
?
```suggestion
        "Bloop version was updated, do you want to restart the running Bloop server?"
```
I'm not sure we need to mention `version`

Reply:
Fixed